app/pb_kits/playbook/pb_currency/currency.rb

```diff
@@ -1,100 +1,46 @@
 # frozen_string_literal: true
 
+require "action_view"
+
 module Playbook
   module PbCurrency
-    class Currency < Playbook::PbKit::Base
-      PROPS = %i[configured_align
-                 configured_classname
-                 configured_data
-                 configured_id
-                 configured_label
-                 configured_value
-                 configured_unit
-                 configured_dollar_sign
-                 configured_size].freeze
-
-      def initialize(align: default_configuration,
-                     classname: default_configuration,
-                     data: default_configuration,
-                     id: default_configuration,
-                     label: default_configuration,
-                     value: default_configuration,
-                     unit: default_configuration,
-                     dollar_sign: default_configuration,
-                     size: default_configuration)
-        self.configured_align = align
-        self.configured_classname = classname
-        self.configured_data = data
-        self.configured_id = id
-        self.configured_label = label
-        self.configured_value = value
-        self.configured_unit = unit
-        self.configured_dollar_sign = dollar_sign
-        self.configured_size = size
-      end
+    class Currency
+      include Playbook::Props
+      include ActionView::Helpers::NumberHelper
 
-      def align
-        align_options = %w[left center right]
-        one_of_value(configured_align, align_options, "left")
-      end
+      partial "pb_currency/currency"
 
-      def dollar_sign
-        pb_dollar_sign = Playbook::PbBody::Body.new(classname: "dollar_sign", color: "light") do
-          default_value(configured_dollar_sign, "")
-        end
-        ApplicationController.renderer.render(partial: pb_dollar_sign, as: :object)
-      end
+      prop :align, type: Playbook::Props::Enum,
+                   values: %w[left center right],
+                   default: "left"
+      prop :amount, required: true
+      prop :label
+      prop :size, type: Playbook::Props::Enum,
+                  values: %w[lg sm],
+                  default: "sm"
+      prop :unit, default: "$"
```

Comment:
The rails and react kits are working differently.

The react kit has 2 props: 
* `currencySymbol` which determines beginning currency symbol ($)
* `unit` which determines the smaller end amount (00)

---

The rails kit has 1 prop of these props
* `unit` which determines the beginning currency symbol ($)
And the smaller end amount is auto-generated without an option to customize.

The unit prop is doing completely different things between the kits. I think these kits should behave the same way.